### ISSUE TITLE
GIF problem in version 3.8.0

### ISSUE THREAD

**Issue by cantalou** [bumptech/glide#1396]
**Glide Version**: 3.8.0

**Integration libraries**: no

**Device/Android Version**: All version

**Issue details / Repro steps / Use case background**: 
Original file: 
![1000285](https://cloud.githubusercontent.com/assets/10027821/17510811/05f3abd0-5e53-11e6-8f22-8a97d31571e7.gif)
Result:
![jdfw](https://cloud.githubusercontent.com/assets/10027821/17510888/5fa6563c-5e53-11e6-877c-dd3699b95ef2.gif) ![image](https://cloud.githubusercontent.com/assets/10027821/17511177/5f6a9240-5e54-11e6-837d-03f996058fd0.png)

I had upgraded to the latest version 3.8.0 but it`s not solved.

**Glide load line / `GlideModule` (if any) / list Adapter code (if any)**:
```java
               Glide.with(context)
                       .load(new GlideUrl(url, mHeader))
                       .diskCacheStrategy(DiskCacheStrategy.SOURCE)
                       .error(error)
                      .placeholder(placeHolder)
                       .into(view);
```

**Layout XML**:
```xml
<ImageView
                        android:id="@+id/activity_personal_homepage_headgear"
                        android:layout_width="145dp"
                        android:layout_height="145dp"
                        android:layout_gravity="center"
                        />
```

**Stack trace / LogCat**:
```ruby
paste stack trace and/or log here
```

**Comment by TWiStErRob** [comment]
I think it may be the same as the bunny in #1094.

**Comment by cantalou** [comment]
HaHa , it works . Tks

**Comment by TWiStErRob** [comment]
What did you change that it works?
#1094 is not fixed yet.

**Comment by jowettcz** [comment]
Hello, TWiStErRob, so which version of glide we should choose if this problem not fixed? BTW,  I cannot find the right way to use the 3.80 snopshot library.
Thank you!

**Comment by TWiStErRob** [comment]
If this gets fixed it'll be tracked in #1094 (that's why this is a duplicate).

This particular issue may not be fixed in snapshot, but it has the best GIF support.
See https://github.com/bumptech/glide/wiki/Snapshots how to use it.
